tests/cupy_tests/statistics_tests/test_order.py

```diff
@@ -117,6 +117,35 @@ def test_percentile_unexpected_interpolation(self, dtype):
             with pytest.raises(ValueError):
                 xp.percentile(a, q, axis=-1, interpolation='deadbeef')
 
+    @testing.for_float_dtypes()
```

Comment:
Would you add a comment to refer to GitHub to show why this test is provided?
```suggestion
    # See gh-4453
    @testing.for_float_dtypes()
```